engines/dfc_provider/spec/requests/offers_spec.rb

```diff
@@ -47,11 +47,30 @@
       }
 
       let(:id) { variant.id }
-      let(:offer) { |example|
+      let(:offer) { offer_example }
+      let(:offer_example) { |example|
         example.metadata[:operation][:parameters].first[:schema][:example]
       }
 
       response "204", "success" do
+        context "with missing stockLimitation" do
+          let(:offer) {
+            offer_example.dup.tap do |o|
+              o.delete(:'dfc-b:stockLimitation')
```

Comment:
Would the absence of a property imply a null value? That seems risky to me: if you sent just one property to be updated, you could inadvertently clear all others. I think it would need to explicitly send a null value.

I think we've had the discussion before, but I forget sorry.

Reply:
Yes, it's not great. And I wouldn't apply this rule in general. But the DFC recommends this for stockLimitation, probably in absence of a better solution. There's no way to express infinity as real value in JSON.

* https://stackoverflow.com/questions/34976378/how-do-you-represent-infinity-in-a-json-api

Reply:
We could propose a new value, of course. Something like our `on_demand` flag. But that can then be confusing in combination with the stock limitation value. :shrug: